app/models/application.rb

```diff
@@ -0,0 +1,18 @@
+class Application < ApplicationRecord
+  has_many :application_pets
+  has_many :pets, through: :application_pets
+
+  validates :name, presence: true
+  validates :street_address, presence: true
+  validates :city, presence: true
+  validates :state, presence: true
+  validates :zip_code, presence: true
+  validates :description, presence: true
+  validates :status, presence: :true, inclusion: {in: ["In Progress", "Accepted", "Rejected", "Pending"]}
+
+  def search_pets_by_name(pet_name)
+    return [] unless pet_name.present?
+
+    Pet.where("name ILIKE ?", "%#{pet_name}%")
```

Comment:
This is a brownfield project, so there's some code already written that you should leverage! Check out the `.search` in the ApplicationRecord file. That's the parent class for all model classes, so you could do `Pet.search` and it would delegate to that parent method! No need to write the query that already exists, just test it